fiona/drvsupport.py

```diff
@@ -197,6 +229,60 @@ def _filter_supported_drivers():
 
 _filter_supported_drivers()
 
+
+@require_gdal_version("2.0")
+def vector_driver_extensions():
+    """
+    Returns
+    -------
+    dict:
+        Map of extensions to the driver.
+    """
+    from fiona.meta import extensions  # prevent circular import
```

Comment:
One thought I had was to add a `_meta.pyx` file and move the method(s) from `ogrext.pyx` there to prevent circular imports. Another option, would be to move these methods to the `meta.py`. Thoughts?